packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -235,15 +226,16 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
             "OptimismPortal: you cannot send messages to the portal contract"
         );
 
-        // Get the output root.
+        // Get the output root. This will fail if there is no
+        // output root for the given block number.
```

Comment:
Note that this function will now succeed for multiple values of `_l2BlockNumber`.